### PR TITLE
DOCS-7722 CI Visibility and Test Visibility Product Landing Page Edits

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do? What is the motivation?

Now that we've separated CI Visibility into CI Pipeline Visibility and Test Visibility, it's time to distinguish the pipeline content from the test content on the product landing pages. We're scrubbing any mention of "tests" from CI Visibility, and making sure that information is covered on the Test Visibility landing page (along with an update to the language/options partial).

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/tests/_index.md [hunk 1]
```diff
@@ -140,19 +137,21 @@ Note: Nested `test.configuration` tags, such as `test.configuration.cpu.memory`,
...
 
-## Use CI tests data
-
-{{% ci-information-collected %}}
-
-### Integrations
+## Enhance your developer workflow
 
-{{< whatsnext desc="Learn about the following integrations with Test Visibility:" >}}
+{{< whatsnext desc="Integrate Test Visibility with tools for reporting code coverage data, enhancing browser tests with Real User Monitoring, and ensuring insights across platforms by streamlining issue identification and issue resolution in your development cycle." >}}
```
Comment (buraizu) [inline content/en/tests/_index.md:142]:
```suggestion
{{< whatsnext desc="Streamline issue identification and resolution in your development cycle by integrating Test Visibility with tools for reporting code coverage data. You can enhance browser tests with Real User Monitoring, and ensure insights across platforms." >}}
```

Comment (buraizu) [reply content/en/tests/_index.md:142]: Left a suggestion here to break up a long sentence, feel free to make any additional adjustments.

**Review by buraizu — APPROVED** [review]
Looks good, approving with a suggestion to break up a long sentence.

**Review by nikita-tkachenko-datadog — APPROVED** [review]

**Review by KickingTheTV — CHANGES_REQUESTED** [review]
Everything LGTM

But can we just make one change while we're here. For the test setup page where we have the table of features and compatibility, can we add a checkmark for Intelligent Test Runner for Ruby? We now support ITR for all languages

**Review by KickingTheTV — APPROVED** [review]
LGTM

**Pull request merged**